styles/Canonical/013-Spell-out-numbers-below-10.yml

```
@@ -0,0 +1,21 @@
+extends: substitution
+message: "Consider using '%s' instead of '%s'"
+link: https://docs.ubuntu.com/styleguide/en#numbers
+scope:
+  - heading
+  - list
+  - sentence
+  - table.header
+  - table.cell
+level: suggestion
+swap:
+  0: zero
+  1: one
+  2 : two
```

The tiniest nitpick: spacing
```suggestion
  2: two
```